### PR TITLE
Create bearing_seal_element.py

### PR DESCRIPTION
**Pull request by rodrigomoliveira1**
This file creates bearing and seal elements.

### CODE REVIEW

File: bearing_seal_element.py [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+            [zpos + hw, -(ypos * 1.3)],
+            [zpos - hw, -(ypos * 1.3)],
+            [zpos - hw, -(ypos * 1.1)],
+            [zpos, -ypos * 1.1],
+        ]
+        ax.add_patch(mpatches.Polygon(seal_points_u, facecolor=self.color))
+        ax.add_patch(mpatches.Polygon(seal_points_l, facecolor=self.color))
+
+
+class IsotSealElement(SealElement):
```
Comment (raphaeltimbo) [inline bearing_seal_element.py]: Remove IsotSealElement for now. We can check later how are we going to add new bearing/seal elements.

File: bearing_seal_element.py [hunk 2]
```diff
@@ -0,0 +1,448 @@
+import warnings
+import numpy as np
+import pandas as pd
+import scipy.interpolate as interpolate
+import matplotlib.pyplot as plt
+import matplotlib.patches as mpatches
+from ross.data_io.read_xl import (
```
Comment (raphaeltimbo) [inline bearing_seal_element.py]: Remove unused imports.

Comment (rodrigomoliveira1) [reply bearing_seal_element.py]: I did both, removed the IsotSealElement class and the unused import. The commit is [here](https://github.com/rodrigomoliveira1/ross/commit/9e5c5d12ec11d74baab8a443c61f502ee4e0369e)

File: bearing_seal_element.py (hunk 1 above)
Comment (rodrigomoliveira1) [reply bearing_seal_element.py]: [Here](https://github.com/rodrigomoliveira1/ross/commit/9e5c5d12ec11d74baab8a443c61f502ee4e0369e) is the related commit for this modification.

**Pull request merged**